server/models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  LastName: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
+    default: "",
+  },
+  availability: [{ type: Number, required: true }],
+});
+
+module.exports = Profile = mongoose.model("profile", profileSchema);
```

Comment:
```suggestion
module.exports = Profile = mongoose.model("Profile", profileSchema);
```
Nit: PascalCase is the standard naming convention for model names in JS